gherkin-utils/java/src/test/java/io/cucumber/gherkin/utils/GherkinPrettyFormatterTest.java

```diff
@@ -0,0 +1,112 @@
+package io.cucumber.gherkin.utils;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class GherkinPrettyFormatterTest {
+
+    public void assertPrettyIdentical(String source) {
+        Assertions.assertEquals(new GherkinPrettyFormatter().format(GherkinParser.parse(source)), source);
+    }
+
+    @Test
+    @DisplayName("Renders a feature with no scenarios")
+    public void renderAFeatureWithNoScenario() {
```

Comment:
With Junit 5 test methods no longer have to be public. Please make them package private.

Reply:
Ok I've already changed it.